src/helpers.ts

```diff
@@ -48,7 +48,8 @@ export function getTestCommand(rootPath: string): string | undefined | null {
  * Checks if the supplied test command could have been generated by create-react-app
  */
 export function isCreateReactAppTestCommand(testCommand: string): boolean {
-  return testCommand && createReactAppBinaryNames.some(binary => testCommand.indexOf(binary + ' test') === 0)
+  testCommand = ` ${testCommand} `
+  return createReactAppBinaryNames.some(binary => testCommand.includes(` ${binary} test `))
```

Comment:
hmm... would it be better to use word-boundary instead of just space? what if the command has quotes, tabs, or semicolon around the test token instead?

a minor performance consideration: if testCommand is falsy (undefined, empty etc), it would be more efficient to short circuit it out of the matching...

Reply:
Or maybe it would be easier to not even check what precedes and succeeds the `react-scripts test` command. `fake-react-scripts tester` would then be a false positive, but that's manageable. We are only doing heuristics and users can customize their settings.

Reply:
yeah, I think this is a reasonable improvement for our current situation...